src/main/java/com/volasoftware/tinder/services/implementations/AuthenticationServiceImpl.java

```diff
@@ -50,11 +53,11 @@ public class AuthenticationServiceImpl implements AuthenticationService {
     @Value("${server.verify-url}")
     private String verifyUrl;
 
-    @Override
-    public List<AccountDto> getAll() {
-        List<Account> accounts = accountRepository.findAll();
+    public List<AccountDto> getAccounts(Pageable pageable) {
+        Page<Account> accountsPage = accountRepository.findAll(pageable);
+        List<Account> accountsList = accountsPage.getContent();
```

Comment:
In your case, the repo will return a small number of records and there will be no problem when you get the content. But what if the records are several thousand? In cases where a Page is used, there must also be some iteration through each possible page, and the page must also have size limits and number.

Reply:
The suggested changes were made.